### ISSUE TITLE
Actions history

### ISSUE THREAD

**Issue by hialvaro** [openblocks-dev/openblocks#174]
I would like to log every action that is being done in OpenBlocks with data such as:

- Users who performed the action
- Previous state before action (what has changed and what was before)
- Date and time the action has been performed
...

Is this currently possible?

**Comment by lmx1117** [comment]
Yes, you can find **Audit logs** here in settings that have recorded all user actions and this is a free trial function only in cloud now, for self-host, we consider making it in enterprise edition.
![image](https://user-images.githubusercontent.com/116335720/211706096-38e70fb5-9665-422c-856c-577e65345081.png)

**Comment by hialvaro** [comment]
Ok. Thank you. I'm using the selfhosted version. Would there be an easy way for me to implement this feature with a selfhosted version while there's no enterprise edition?

**Comment by neon-balcony** [comment]
Hey @hialvaro , a brand new enterprise image would be released soon, it should be ready by the end of Jan 2023 😄

**Comment by raunak6662** [comment]
Hey, what is the price of open blocks in enterprise version. And one issue I faced while using selfhosted version is that it is not able to import the json file exported by retool.

**Comment by TensorDog** [comment]
> 
Hi,  i can communicate with you about Openblokcs enterprise edition feature and the price. Would you like to chat in our discord community. My discord ID is Fan#1527
https://discord.com/invite/z5W2YHXdtt
